arangod/GeneralServer/HttpCommTask.cpp

```diff
@@ -295,7 +297,8 @@ bool HttpCommTask<T>::readCallback(asio_ns::error_code ec) {
 
 template <SocketType T>
 void HttpCommTask<T>::setIOTimeout() {
-  double secs = GeneralServerFeature::keepAliveTimeout();
+  auto& gs = this->_server.server().template getFeature<GeneralServerFeature>();
```

Comment:
Zhis (and the following usage of getFeature in `sendResponse`) will perform the hashing and lookup of the GeneralServerFeature every time a timeout is set. I don't have a good feeling how often this happens during the lifetime of a request/response cycle. If it happens often, it could be more efficient to perform the lookup just once, in the ctor of the CommTask, and from then on keep a reference to the looked-up feature ready for usage.